src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,41 @@
+package coffee.order;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class CoffeeOrderBoard {
+
+  private int lastIndex = 0;
+  Queue<Order> orders = new LinkedList<>();
+
+  public void add(String customerName) {
+    lastIndex++;
+    orders.add(new Order(lastIndex, customerName));
+  }
+
+  public void deliver() {
+    orders.poll();
+  }
+
+  public void deliver(int index) {
+    Iterator<Order> iterator = orders.iterator();
+    while (iterator.hasNext()) {
+      Order order = iterator.next();
+      if (order.getOrderId() == index) {
+        iterator.remove();
+        return;
```

Comment:
it is delivered and the method returns void. probably it is a good idea to return the Order that were removed